third_party/dcurl/BUILD.bzl

```diff
@@ -0,0 +1,101 @@
+licenses(["notice"])  # MIT license
+
+config_setting(
+    name = "avx",
+    values = {"define": "BUILD_AVX=1"},
+)
+
+config_setting(
+    name = "stat",
+    values = {"define": "BUILD_STAT=1"},
+)
+
+ISA = select({
+    "avx": ["-mavx -DENABLE_AVX"],
+    "//conditions:default": ["-msse2 -DENABLE_SSE"],
```

Comment:
Remove this line.

Reply:
Does this mean remove ISA variable or only the default condition?